src/main.rs

```diff
@@ -67,6 +66,9 @@ extern "C" {
     static _stack_end: u8;
 }
 
+// stash the vector state for use deeper in the vm_cpu code
+static VECTORS_ENABLED: Once<bool> = Once::new();
+
```

Comment:
Do we really need this? It only gets used once, so I'm not sure it's that useful of a shortcut vs `CpuInfo::get().has_vector()`

Reply:
didn't realize that CPU_INFO was static. will fix